projects/samples/contests/robocup/protos/RobocupSoccerField.proto

```diff
@@ -0,0 +1,338 @@
+#VRML_SIM R2021a utf8
+# license: Apache License 2.0
+# license url: http://www.apache.org/licenses/LICENSE-2.0
+# tags: static
+# Official soccer field for the Robocup Humanoid League
```

Comment:
The final point is missing but for consistency with the other PROTO descriptions I would also add the year.
```suggestion
# Official soccer field for the Robocup Humanoid League 2021.
```